AzureVMPrivIp.bicep

```diff
@@ -0,0 +1,150 @@
+// az deployment group create --template-file AzureVM.bicep --resource-group "test_group"
```

Comment:
I think we can do this without introducing another Bicep module: https://github.com/Azure/mock-spacestation/compare/glenn/useExistingNetwork